.lintstagedrc.js

```diff
@@ -8,4 +8,5 @@ const buildEslintCommand = filenames =>
 module.exports = {
   '*.{js,jsx,ts,tsx}': [buildEslintCommand],
   '*': ['yarn pretty:write'], // .prettierrc takes care of filtering files.
+  '*.{js,jsx,ts,tsx}': ['yarn test:all'],
```

Comment:
Good with this for now, but I think once we have a CI pipeline set up, we can just do it there to save time on commits.